build.env

```diff
@@ -16,7 +16,7 @@ BASE_IMAGE=docker.io/ceph/ceph:v16
 CEPH_VERSION=octopus
 
 # standard Golang options
-GOLANG_VERSION=1.16.4
+GOLANG_VERSION=1.17.4
```

Comment:
This obviously is part of #2703 already, so a rebase will be needed. Setting `depends-on: #2703` in the description so that Mergify should pick it up automatically.